test/actions/AuthorisedActionSpec.scala

```diff
@@ -349,41 +343,8 @@ class AuthorisedActionSpec extends ControllerUnitTest
         }
       }
 
-      "results in an Exception other than an AuthException error being returned for Primary Agent check" should {
```

Comment:
Looking again at this test I think the reason it might have been failing is due to the comment I left on AuthorisedAction about the e case needing to be moved outside of the recover block